src/screens/BB-BUS/DetailScreen.js

```diff
@@ -157,7 +168,9 @@ export const DetailScreen = ({ route }) => {
         }
       >
         {!!forms?.length && (
-          <View style={styles.formContainer}>{forms.map((form) => renderForm(form))}</View>
+          <View style={styles.formContainer}>
+            {forms.map((form) => renderForm(headerTitle, form, rootRouteName))}
```

Comment:
why do we need the `rootRouteName` here. it is an optional param.

Reply:
```
  const rootRouteName = route.params?.rootRouteName ?? '';
```
Yes, it is not required, but we already have `rootRouteName` as param. That's why I added it.

Reply:
ok thanks, keep it.